net-mgmt/pfSense-pkg-arpwatch/files/usr/local/pkg/arpwatch.inc

```diff
@@ -62,7 +62,15 @@ function arpwatch_install_command() {
 
 			touch($arp_file);
 
-			$rc['start'] .= "/usr/local/sbin/arpwatch $disable_carp $disable_bogons $disable_zero $enable_zeropad -f $arp_file -i $ifname -w $notifications_recipient"."\n";
+			$rc['start'] .= "/usr/local/sbin/arpwatch" 
+			    ." $disable_carp"
```

Comment:
I didn't spot this before but these should be changed on way or the other as follows:
* Put `{}` around the variable name, so: `. " {$disable_carp}"`
-or-
* Take the variable out of the quotes:  `. " " . $disable_carp`
Given the structure of how it's been changed, the second form is probably better.

Reply:
No worries.

Also, no one every accused me of being a php expert.

I believe I've converted the code to use the variables outside of strings or to curly brace quote variables inside strings.